serenata_toolbox/federal_senate/dataset.py

```diff
@@ -14,6 +14,12 @@ class Dataset:
     AVAILABLE_YEARS = [year for year in range(2008, date.today().year + 1)]
 
     def __init__(self, path, years=AVAILABLE_YEARS):
+        if not os.path.isdir(path):
+            try:
+                os.mkdir(os.path.join(path))
+            except OSError:
+                log.error('Couldn\'t create the output directory')
+                raise
```

Comment:
We should be explicit about which exception we're raising here. We can either throw the default `OSError` or build our custom exception.

TBH I would simply and just trust the `OSError`:

```python
if not os.path.isdir(path):
    os.mkdir(os.path.join(path))
```

With no `try`/`exception` at this point. What do you think about it?